client/src/partials/journal/journal.js

```diff
@@ -58,37 +66,115 @@ function JournalController(Transactions, Sorting, Grouping, Pagination, Filterin
    *    pagination = new Pagination(vm.gridOptions, Transactions.list.data);
    *    grouping = new Grouping(vm.gridOptions);
    *    filtering  = new Filtering();
+   *    columnConfig = new ColumnConfig(vm.gridOptions);   *    
    * Note:
    *   Setting the grouping priority without sorting by the same column will
    *   cause unexpected behaviour (splitting up of groups) when sorting
    *   other columns. This can be avoided by setting default sort and group.
+   *  @todo using ui-grid internazation if necessary for columns label
    */
-  vm.gridOptions.columnDefs = [
-    { field : 'trans_date', displayName : 'Date', cellFilter : 'date:"mediumDate"', filter : { condition : filtering.byDate } },
-    { field : 'description', displayName : 'Description' },
-    { field : 'account_number', displayName : 'Account' },
-    { field : 'debit_equiv', displayName : 'Debit' },
-    { field : 'credit_equiv', displayName : 'Credit' },
-    { field : 'trans_id',
-      displayName : 'Transaction',
-      sortingAlgorithm : sorting.transactionIds,
-      sort : { priority : 0, direction : 'asc' },
-      grouping : { groupPriority : 0 }
-    },
-
-    // @todo this should be formatted as a currency icon vs. an ID
-    { field : 'currency_id', displayName : 'Currency', visible: false },
-
-    // @todo this should be formatted showing the debitor/credior
-    { field : 'deb_cred_uuid', displayName : 'Recipient', visible : false },
-
-    // @fixme inv_po_id -> reference
-    { field : 'inv_po_id', displayName : 'Reference Document', visible : false },
-    { field : 'user', displayName : 'Responsible', visible : false },
-    { field : 'period_summary', displayName : 'Period', visible : false },
-
-    // @fixme this field should not come from the database as 'cc'
-    { field : 'cc', displayName : 'Cost Center', visible : false }
-  ];
+
+    columns = [
```

Comment:
Many of these fields no longer exist in the new journal backed schema.  Since you are working with the columns, could you update to use the new schema?